starter-code/pizza.js

```diff
@@ -1 +1,68 @@
 // Write your Pizza Builder JavaScript in this file.
+
+var totalPrice = $("strong").html(13);
+var newPrice =0;
+
+//Pepperonni button
+$(".btn-pepperonni").click(function () {
+     $(".pep").toggle();
+     $(".price li:contains(pepperonni)").toggleClass("not-displayed");
+    //  if ($(".btn-pepperonni").hasClass("active")) {
+    //  } else {
+    //      newPrice = Number(totalPrice) - 3;
+    //      $("strong").html(newPrice);
+    //  }
+});
+
+//Mushroom button
+$(".btn-mushrooms").click(function () {
+    $(".mushroom").toggle();
+    $(".price li:contains(mushroom)").toggleClass("not-displayed");
```

Comment:
You can do this whithout having to touch to the css file by doing the same thing as you are doing just above : 
$(".price li:contains(mushroom)").toggle();